jest.config.js

```diff
@@ -14,6 +15,9 @@ module.exports = {
     '<rootDir>/src/hooks/**/*.{ts,tsx,js,jsx}',
     '<rootDir>/src/styles/**/*.{ts,tsx,js,jsx}',
     '!<rootDir>/src/styles/overrides/*.{ts,tsx,js,jsx}',
+    '!<rootDir>/src/components/index.ts',
+    '!<rootDir>/src/styles/index.ts',
+    '!<rootDir>/src/components/**/index.ts',
```

Comment:
We use the "barrel pattern", where the `index` file is always just a re-export of the components.  So for example:

`src/components/Button`
`src/components/Button/Button.tsx`
`src/components/Button/index.ts`

The `index` file just re-exports functions and components from `Button.tsx`:

```js
export { Button, ButtonProps, ButtonStylesKey, ButtonClasses } from './Button';
```

Due to this pattern, it's safe to remove these from collected coverage imo